src/dft.cpp

```diff
@@ -174,7 +174,30 @@ dft_chunk *fields::add_dft(component c, const volume &where, const double *freq,
   dft_chunk_data data;
   data.c = c;
   data.vc = vc;
+
+  if (decimation_factor == 0) {
+    double tol = 1e-7;
```

Comment:
Should this be `1e-7` in single precision and `1e-15` in double precision?

Reply:
On the other hand, that would make it harder to directly compare single and double precision…